docs/README.md

```
@@ -0,0 +1,49 @@
+
+We have now migrated document dependancy management and build to poetry
+based.
+
+Building should be the same for all platforms. These instructions assume
+you already have a poetry environment setup, as per normal development.
+
+The docs dependancies are managed in the [pyproject.toml]{.title-ref}
+file. To install the docs dependancies, run the following command:
+
+``` console
+$ poetry install --only docs
```

Markdownlint (0.35.0)

Dollar signs used before commands without showing output

(MD014, commands-show-output)